src/theme/theme.ts

```diff
@@ -58,33 +62,49 @@ const LightMaterialUITheme = createTheme(theme, {
             xl: 1536,
         },
     },
-    typography: typography(theme),
-    shadows: shadows(theme),
-    components: {
-        ...buttons(theme),
-        /**
-         * Styles below adjust MUI TablePagination component to the current wep-app style
-         * Should be removed after moving explorer to the MUI Data Grid
-         */
-        MuiTablePagination: {
-            styleOverrides: {
-                toolbar: {
-                    "@media (min-width: 0px)": {
-                        minHeight: "30px",
-                        paddingLeft: 0,
-                    },
-                    fontSize: "12px",
-                },
-                displayedRows: {
-                    margin: 0,
-                    fontSize: "12px",
-                },
-                actions: {
-                    marginLeft: 0,
+};
+
+/**
+ * Styles below adjust MUI TablePagination component to the current wep-app style
+ * Should be removed after moving explorer to the MUI Data Grid
+ */
+const MUITablePaginationSettings = {
```

Comment:
I would rather move these styles to web-app, in that case we won't forget to remove them.

Reply:
Probably. MUI and data grid looks like something you're working on right now, so I left it effectively intact.

Reply:
@zoomchik please make a note of this in one of your tasks